infection_monkey/config.py

```diff
@@ -1,11 +1,12 @@
 import os
+import struct
 import sys
 import types
 import uuid
 from abc import ABCMeta
 from itertools import product
 
-from exploit import WmiExploiter, Ms08_067_Exploiter, SmbExploiter, RdpExploiter, SSHExploiter, ShellShockExploiter, \
+from exploit import WmiExploiter, SmbExploiter, SSHExploiter, ShellShockExploiter, \
```

Comment:
You can't remove these imports becuase then using the exploiters will crash (Python won't know of these exploiters)